torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -2670,6 +2674,11 @@ def test_scatter(self):
             group, group_id, rank = self._init_global_test()
             self._test_scatter_helper(group, group_id, rank)
 
+        @unittest.skipIf(BACKEND == "nccl", "Nccl does not support scatter")
+        def test_scatter_complex(self):
+            group, group_id, rank = self._init_global_test()
+            self._test_scatter_helper(group, group_id, rank, dtype=torch.cfloat)
```

Comment:
what's the difference between `torch.cfloat` and `torch.complex64`?

Reply:
They are both representing 64-bit complex data type.